docs/setup/access.asciidoc

```diff
@@ -37,4 +37,22 @@ image::images/kibana-status-page-7_14_0.png[Kibana server status page]
 
 For JSON-formatted server status details, use the `localhost:5601/api/status` API endpoint. 
 
+[float]
+[[not-ready]]
+=== {kib} not ready
+
+During install or post upgrade, you may infrequently encounter the Kibana start up error
```

Comment:
Suggest replacing lines 44 through 50 with this:

If you receive an error that the {kib} server is not ready, check the following.